### PR TITLE
Add simple test for SequenceMapExp

### PR DESCRIPTION
**Pull request by orlade-anz**
WIP #356.

Changes proposed in this pull request:
- Add a basic test for SequenceMapExpr which previously had none.

Checklist:
- [x] Added related tests

### CODE REVIEW

**Comment by nofun97** [comment]
so we test expressions in the `rel` package because we can just write the arrai code for the expression, this is for example https://github.com/arr-ai/arrai/blob/master/rel/expr_indexed_seqmap.go

**Comment by orlade-anz** [comment]
Also WIP #381

File: rel/expr_seqmap_test.go [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import (
+	"testing"
+
+	"github.com/arr-ai/wbnf/parser"
+)
+
+func TestSequenceMapExprAccessors(t *testing.T) {
+	t.Parallel()
+
+	lhs := NewArray(NewNumber(float64(1)))
```
Comment (nofun97) [inline rel/expr_seqmap_test.go:12]:
```suggestion
	lhs := NewArray(NewNumber(1))
```

File: rel/expr_seqmap_test.go [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+	fn := NewNumber(float64(2))
```
Comment (nofun97) [inline rel/expr_seqmap_test.go:13]:
```suggestion
	fn := NewNumber(2)
```
